streaming/pom.xml

```diff
@@ -126,19 +126,43 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-exec</artifactId>
+      <version>${project.version}</version>
+      <type>test-jar</type>
+      <scope>test</scope>
+    </dependency>
     <dependency>
       <groupId>org.apache.hive</groupId>
       <artifactId>hive-standalone-metastore-server</artifactId>
       <version>4.1.0-SNAPSHOT</version>
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-common</artifactId>
+      <version>${project.version}</version>
+      <classifier>tests</classifier>
+      <scope>test</scope>
+    </dependency>
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-mapreduce-client-jobclient</artifactId>
       <version>${hadoop.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.tez</groupId>
+      <artifactId>tez-mapreduce</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.tez</groupId>
+      <artifactId>tez-dag</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Why is this required?

Reply:
tez-mapreduce: MRInput (as mentioned above)
tez-dag: I cannot recall I'll double-check
hive-exec: I cannot recall I'll double-check
hive-common: HiveConfForTest

Reply:
without tez-mapreduce it fails, it's needed
```
java.lang.NoClassDefFoundError: org/apache/tez/mapreduce/hadoop/InputSplitInfo
	at org.apache.hadoop.hive.ql.plan.TezWork.<init>(TezWork.java:91)
	at org.apache.hadoop.hive.ql.parse.GenTezProcContext.<init>(GenTezProcContext.java:176)
	at org.apache.hadoop.hive.ql.parse.TezCompiler.generateTaskTree(TezCompiler.java:634)
	at org.apache.hadoop.hive.ql.parse.TaskCompiler.compile(TaskCompiler.java:302)
...
Caused by: java.lang.ClassNotFoundException: org.apache.tez.mapreduce.hadoop.InputSplitInfo
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	... 67 more
```

without tez-dag:
```
Caused by: java.lang.ClassNotFoundException: org.apache.tez.dag.app.TezTaskCommunicatorImpl
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382) ~[?:1.8.0_292]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[?:1.8.0_292]
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352) ~[?:1.8.0_292]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_292]
	... 83 more
```

hive-exec: removed, it was a leftover from early versions of this patch when HiveConfForTest was in the hive-exec module

hive-common: needed because of HiveConfForTest
without it:
```[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M4:test (default-test) on project hive-streaming: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M4:test failed: org.junit.platform.commons.JUnitException: TestEngine with ID 'junit-jupiter' failed to discover tests: ClassSelector [className = 'org.apache.hive.streaming.TestStreamingDynamicPartitioning'] resolution failed: org/apache/hadoop/hive/conf/HiveConfForTest: org.apache.hadoop.hive.conf.HiveConfForTest -> [Help 1]
```